test/Greeter3.test.js

```diff
@@ -0,0 +1,29 @@
+const Greeter3= require('../src/Greeter3');
+
+//  test('description', ()=>{
```

Comment:
czemu wyłączone?

Reply:
bo z nieznanych przyczyn nie pójdzie, to znaczy przez konstruktor. Jaki paramter miałby być na początku?

Reply:
kombinuj, klient zgłasza buga :D